### ISSUE TITLE
Server hangs and crashes while generating new dimension

### ISSUE THREAD

**Issue by timfennis** [Qmunity/BluePower#414]
I initially reported this issue over at the RFTools page but according to McJty the issue could be related to BluePower which is why I'm posting it here now.

**What where you doing when you had the issue?**
Generating a new RFTools dimension. I've added quite a lot of information in my original report over there: https://github.com/McJty/RFTools/issues/349

**What did you expect to happen?**
I expected the dimension to create itself properly. When I forced myself in to the dimension I expected to find issues with the receiving end of the teleporter.

**What happened instead?**
First my client gets Timed out and I disconnect. Logging in to the server shows that I'm using 100% CPU and eventually the console output of the server shows an out of memory error.

**Crashlog**
I get multiple lines like the one below that indicate to me that generating a crash report is impossible. 

```
[22:56:37] [Server thread/INFO] [STDOUT]: [net.minecraft.crash.CrashReport:func_85057_a:320]: Negative index in crash report handler (32/1020)
```

The crash report directory in my server does not contain any crash reports. The log directory contains some .json crash reports like the one I've uploaded here http://pastebin.com/tLLADxFz

Below I have a couple of pastebins from the console output of my server around the time I tried to enter/create the dimensions. (These contain clues that bluepower might be involved)

http://pastebin.com/bYpTv9YA (2nd attempt at loading dimension)
http://pastebin.com/05Vd7BPh (3rd attempt at loading dimension)

**Steps to reproduce**
I have a backup of my server (around 300MB) where my character is in the corrupted dimension that instantly causes the server to crash upon connecting to it. If I move my character back to the overworld with MCEdit everything works fine. On that backup I also have the dimlet that created the corrupt dimension that could possibly provide useful tracking down the issue. I'm able to share the server backup with you upon request.

**Comment by Quetzi** [comment]
Going to need a full log from the server to have any idea what is going on here, sorry.

**Comment by timfennis** [comment]
Sure thing, I'll upload the log files form yesterday evening first thing when I get home tonight.

**Comment by timfennis** [comment]
Alright I've opened up my log directory and the relevant data seems to be in 3 different files:

`fml-server2.log` 49,877KB
`fml-server3.log`  2,744KB
`fml-server-latest.log` 2,743KB

The 2nd file seems to contain the first crash with no information related to bluepower. The 3rd file contains another crash also without information related to bluepower and the `fml-server-latest.log` file contains the stacktrace with bluepower in it.

Becase all of these files exceed the maximum filesize allowed on pastebin I've compressed my whole log directory and uploaded it here http://www.timfennis.com/logs.7z and here http://www.timfennis.com/logs.zip

I'll copy this post over in the RFTools repo.
